src/Bowling.js

```diff
@@ -0,0 +1,92 @@
+'use srict'
+
+// var Frame = require('../src/frame')
+function Frame (first, second) {
+  this.rolls = [first, second]
+  this.MAX_POINTS = 10
+  this.spare = false
+  this.strike = false
+}
+
+Frame.prototype = {
+  isStrike: function () {
+    if (this.first === this.MAX_POINTS) {
+      this.strike = true
+    }
+  },
+  isSpare: function () {
+    if (this.first + this.second === this.MAX_POINTS) {
+      this.spare = true
+    }
+  },
+  firstRoll: function () {
+    return this.rolls[0]
+  },
+  secondRoll: function () {
+    return this.rolls[1]
+  },
+  rollScore: function () {
+    return this.first + this.second
+  }
+}
+
+function Bowling () {
+  this.frames = []
+  this.totalPoints = 0
+  this.MAX_FRAMES = 10
```

Comment:
I mentioned in a comment on your tests that we could go from specific to general - to highlight what I mean by that, consider whether you could inject a variable maxFrames here.